src/modules/python.rs

```diff
@@ -35,25 +39,6 @@ pub fn segment(context: &Context) -> Option<Module> {
     }
 }
 
-fn has_py_files(dir_entry: &PathBuf) -> bool {
```

Comment:
Feels good to strip this away 🤤

Reply:
![image](https://user-images.githubusercontent.com/6132021/57106411-4747f500-6cfb-11e9-9301-546d01969c62.png)